packages/components/src/components/alert/alert.tsx

```diff
@@ -454,17 +454,18 @@ export class Alert extends LitElement {
 
   private renderCloseButton(): JsxNode {
     return (
-      <button
-        ariaLabel={this.messages.close}
-        class={CSS.close}
-        key="close"
-        onClick={this.closeAlert}
-        onFocusIn={this.autoClose ? this.handleKeyBoardFocus : null}
-        onFocusOut={this.autoClose ? this.handleKeyBoardBlur : null}
-        type="button"
-      >
-        <calcite-icon icon="x" scale={getIconScale(this.scale)} />
-      </button>
+      <div class={CSS.close}>
```

Comment:
Is the wrapper needed or can this return an action with `.close` applied?

Reply:
Great idea, it works fine without the wrapper.  Thanks!